common/models.py

```diff
@@ -131,7 +132,8 @@ class RebaseData(BaseModel):
 class BackportData(BaseModel):
     """Data for backport resolution."""
     package: str = Field(description="Package name")
-    patch_url: str = Field(description="URL to the source of the fix that was validated using PatchValidator tool")
+    patch_urls: list[str] = Field(
+        description="A list of URLs to the sources of the fixes that were validated using the PatchValidator tool")
```

Comment:
`upstream_patches` is a list but it's not treated as one in the prompt template. I can see that a direct substitution renders as e.g. `['patch1', 'patch2']`, which is probably fine for the models to read, so take this just as a FYI. The templating docs are here: https://mustache.github.io/mustache.5.html

Reply:
I thought you meant the `<>` syntax in system prompt, not the prompt templating, now I understand